compiler/quilt/vfs.py

```diff
@@ -106,31 +106,73 @@ def open_func(filename, mode='r', module=module, *args, **kwargs):
     patcher.start()
     return patcher
 
+
+# Note: To test this, uninstall h5py, then run your quilt_vfs_test.py.
```

Comment:
easier to just manually set the module mappings, then include a non-existent module ?

Reply:
Yep. >.>